app/views/submissions/_submissions.html.haml

```diff
@@ -39,7 +39,8 @@
 
           - if @ontology.admin?(session[:user])
             - r.td { raw sub.submissionId }
-          - r.td { raw status_link(sub, sub.submissionId==submission_readyId)}
+          - r.td do
```

Comment:
you don't need to add a column for the status, why not just append it to the version column?

Reply:
I tried that approach, but it caused an issue

Reply:
OK, it's not important.